Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -668,3 +668,35 @@ def onNodeAdded(caller, event, calldata):
   slicer.mrmlScene.RemoveObserver(sceneObserverTag)
 
   return loadedNodeIDs
+
+def loadFromDICOMWeb(dicomWebEndpoint, studyUID, seriesUID="", accessToken=""):
```

Comment:
Add method documentation. Preferably also include an example of how it can be used.